geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/FunctionContextArgumentProvider.java

```diff
@@ -60,7 +62,7 @@ String getMember(FunctionContext<?> context) {
    */
   XmlEntity createXmlEntity(FunctionContext<?> context) {
     return new XmlEntity(createCacheProvider(context), CACHE, PREFIX, NAMESPACE,
-        CONNECTION_SERVICE.getTypeName());
+        CONNECTION_SERVICE.getTypeName(), NAME, CONNECTION_SERVICE.getTypeName());
```

Comment:
This is confusing, are we saying the parent and child types are same?

Reply:
Yes. XmlEntity has a flawed and confusing design. Someone needs to redo that class.